extensions/azurecore/src/test/azureResource/tree/accountTreeNode.test.ts

```diff
@@ -48,7 +48,8 @@ const mockAccount: azdata.Account = {
 		displayName: 'mock_account@test.com',
 		accountType: 'Microsoft',
 		contextualDisplayName: 'test',
-		userId: 'test@email.com'
+		userId: 'test@email.com',
```

Comment:
How about giving this an oid (whatever that looks like) instead so it's clear this is supposed to be different?